---
title: "Title here"
author: "Travis Blimkie"
date: "`r format(Sys.time(), '%B %d, %Y')`"
output:
  html_document:
    theme: readable
    highlight: kate
    fig_caption: yes
    code_folding: show
    toc: yes
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(
  echo    = TRUE, 
  message = FALSE, 
  warning = FALSE, 
  comment = ""
)
```

***

## Load R Packages
```{r libraries}
suppressPackageStartupMessages({
	library(DESeq2)
	library(tRavis)
	library(tidyverse)
})
```

<br>

## Overview


<br>

## Quality Assessment
The typical pipeline for processing RNA-Seq samples is as follows:

- Check quality of fastq reads with [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
- Aligned reads to the reference genome of choice with [STAR](https://github.com/alexdobin/STAR)
- Count aligned reads using [HTSeq's count](https://htseq.readthedocs.io/en/release_0.11.1/count.html) function

The plots below are taken from the report compiled by [MultiQC](https://multiqc.info/).

### FastQC - Read Files
<!-- ![](../multiqc_data/fastqc_per_base_sequence_quality_plot.svg) -->

<br>

### STAR - Alignments
<!-- ![](../multiqc_data/star_alignment_plot.svg) -->

<br>

### HTSeq - Gene Counts
<!-- ![](../multiqc_data/htseq_assignment_plot.svg) -->

<br>

## Importing Counts
```{r count_matrix}

```

<br>

## Library Sizes
```{r library_table}

```

<br>

## Create DESeq Objects
```{r deseq}

```

<br>

## Principle Component Analysis
```{r pca, fig.width = 10, fig.height = 5, fig.align = "right"}

```

<br>

## Differential Expression
```{r degenes}

```

<br>

## Functional Analysis
```{r functional_analysis}

```

<br>

***

## Version Information
- FastQC x
- STAR x
- htseq-count x
- MultiQC x

```{r sessionInfo, echo = FALSE}
sessionInfo()
```
